log.go

```diff
@@ -47,21 +47,21 @@ import (
 //
 // Loggers can not be used before the log rotator has been initialized with a
 // log file.  This must be performed early during application startup by
-// calling logWriter.InitLogRotator.
+// calling LogWriter.InitLogRotator.
 var (
-	logWriter = build.NewRotatingLogWriter()
+	LogWriter = build.NewRotatingLogWriter()
```

Comment:
doesn't appear like this is ever used externally?

Reply:
I think this is for external callers to hook into the existing logging for `lnd`?

Reply:
Yes, we need to be able to hook all logging subsystems of the external subservers into the main log writer. I moved this static instance to the `build` package too as it makes more sense there IMO.